src/measure_theory/interval_integral.lean

```diff
@@ -652,6 +660,18 @@ lemma integral_add_adjacent_intervals (hab : interval_integrable f μ a b)
   ∫ x in a..b, f x ∂μ + ∫ x in b..c, f x ∂μ = ∫ x in a..c, f x ∂μ :=
 by rw [← add_neg_eq_zero, ← integral_symm, integral_add_adjacent_intervals_cancel hab hbc]
 
+lemma integral_sum_adjacent_intervals {a : ℕ → α} {n : ℕ}
```

Comment:
`integral_sum` or `sum_integral`?

Reply:
I kept the same order as in `integral_add_adjacent_intervals` but it definitely makes more sense the other way around.